src/pillowtalk_impl.c

```diff
@@ -1040,3 +1080,5 @@ static void generate_node_json(pt_node_t* node, yajl_gen g)
     yajl_gen_null(g);
   }
 }
+
+/* vim: set tabstop=2 shiftwidth=2: */
```

Comment:
Well, regarding coding style, honestly, this package needs some work.  Ideally, we could use editorconfig and also automatically apply a chosen coding convention (e.g. GNU C).